config/v1/types_infrastructure.go

```diff
@@ -1104,6 +1136,14 @@ type IBMCloudPlatformStatus struct {
 	// DNSInstanceCRN is the CRN of the DNS Services instance managing the DNS zone
 	// for the cluster's base domain
 	DNSInstanceCRN string `json:"dnsInstanceCRN,omitempty"`
+
+        // ServiceEndpoints is a list of custom endpoints which will override the default
+        // service endpoints of an IBM Cloud service. These endpoints are consumed by
+	// components within the cluster to reach the respective IBM Cloud Services.
```

Comment:
This final sentence isn't present on the spec version, should copy it over

Reply:
Okay, I'll copy it, assuming I keep Spec (pending your thoughts on removing Spec until CCM supports Spec to Status validation).